documentation/add/07-sqw_redesign.md

```diff
@@ -130,28 +156,23 @@ Image pixel data is generated from the `PixelData` via one or more projections.
 | signal[] | Mean intensity, calculated as `Sum(pix_signal(k))/npix(k)` | (1), (2) |
 | err[] | Average error, calculated as  `sqrt(Sum(pix_variance(k)/npix(k)))` |(1), (2) |
 | npix[] | Number of detector pixels contributing to each image pixel ||
-| uoffset[] | Offset of pixel projection axes origin | (3) |
-| u_to_rlu \[\]\[\] | Matrix of pixel projection in hkle | (3) |
-| ulen[] | Length of pixel projection axes Ang^-1 or meV | (3), (4) |
+| proj | The instance of `aProjection` class, describing the transformation from Crystal Cartesian to Image coordinate system | |
+| axes | The instance of `axes_block` class, defining the shape and size of *signal* *err* and *npix* arrays and the units along their axes |(3) |
 
-The `Axis` classes describes image axes
-- value range
-- unit vectors
-- units
-- matrix mapping these axes to the pixel data
-- requires a well-defined mapping from image pixels to source data pixel
 
 
 **Notes**
 (1): if the image data is updated, e.g. after a slice or projection, the backing pixel data must be updated/reordered
 (2): `pix_signal` represents the array of pixel signal data from which this image data was derived, `pix_variance` the array of pixel variance.
-(3): uoffset and ulen are 4x1 vectors and u_to_rlue is a 4x4 matrix, with the four coordinates are always assumed to be (u1, u2, u3, dE) in that order.
-(4): u1, u2, u3 have units of Ang^-1, dE has units of meV.
+(3): The `axes_block` classes describes image axes
+- value range
+- unit vectors
+- units
 
-### data_sqw_dnd class
-The replacement for **data_sqw_dnd** class (current implementation of the image) is based on generic projection transformation and the methods, necessary to implement generic cut procedure.:
 
-Suggested Image block ( **data_sqw_dnd**) consists of 1) an **axes_block** class, defining image binning ranges and coordinates axes used in plotting, 2) a particular instance of **aProjection** class defining the transformation from Crystal Cartesian coordinate system of the **PixelData** class into Image coordinate system (e.g. hkl-dE coordinates for rectilinear projection) and back and 3) *signal*, *error* and *npix* arrays, having the dimensions defined by the **axes_block** and containing information about the pixels, contributed into appropriate bins of the **axes_block**.
+The **DnD class** (current implementation of the image) consists of generic projection transformation and the methods, necessary to implement generic cut procedure.:
```

Comment:
this - here L.173 and also L.175 - is very important. We are saying that the DnD as well as containing the image also contains the generic projection. Maybe this should be a separate class. For sqw's it will also influence the choice of pixels, which are outside this class. For powder average, does this average pixels or only image voxels? I think this is worth more explanation

Reply:
I have reformatted this statement a bit in a hope to make it clearer but I fully insist on its meaning. I do not see any need in  other separate classes to do projection.